docs/modules/plugins/pages/plugin-mobile-app.adoc

```diff
@@ -475,4 +475,36 @@ When I change Appium session settings:
 |allowInvisibleElements|true |
 ----
 
+=== Select date in date picker
+
+==== *_Info_*
+
+Selects a next or previous picker wheel value in date picker
+
+[WARNING]
+Supported only for `iOS` platform
+Located by the locator element type must be `XCUIElementTypePickerWheel`
```

Comment:
Only `iOS` platform is supported and the target element must be of type `XCUIElementTypePickerWheel`

Reply:
these are two separate statements, imho it's more clear to describe them like this

Reply:
then convert the statements to the list